tests/sources/json/test_aardvark.py

```diff
@@ -256,6 +256,48 @@ def test_aardvark_get_rights_success(aardvark_record_all_fields):
         ),
     ]
 
+    aardvark_record_all_fields_mit_restricted = next(aardvark_record_all_fields)
```

Comment:
These should be separate tests, more and smaller tests are much more preferable to me than an aggregate test that has several reasons to fail re: Clean Code

Reply:
Change has been made. Please see the latest commit.